### ISSUE TITLE
Fine-tuning the hyperparameters

### ISSUE THREAD

**Issue by ayush2051** [bloomsburyai/question-generation#23]
Hi Tom,
Did you fine-tune any of the hyper-parameters that u have shared in the FLAGS file as I tried with the same and got a BLEU score of 14.17, NLL- 40.33.; with the GLOVE vocab, PG set to FALSE. Can u let me know the exact split of the dev and test set samples that you used for the evaluation as this high score may be biased due to some different split I took?

**Comment by tomhosking** [comment]
Hi, I'd strongly recommend using the split from here as it has become the standard split in the literature:
https://github.com/tomhosking/squad-du-split

I didn't do a full hyperparameter optimisation, but did play around with them a bit during development.

**Comment by ayush2051** [comment]
Thanks a lot Tom!! will try a few combinations on the top of that.
